tests/experimental/datasources/test_pandas_datasource.py

```diff
@@ -119,7 +121,13 @@ class TestDynamicPandasAssets:
                 "read_table",
                 marks=pytest.mark.xfail(reason="conflict with 'table' type name"),
             ),
-            param("read_xml"),
+            param(
+                "read_xml",
+                marks=pytest.mark.skipif(
+                    PANDAS_VERSION.startswith("1.1"),
+                    reason=f"read_xml does not exist on {PANDAS_VERSION} ",
```

Comment:
Wasted way too much time on this 😆 🙃 
https://pandas.pydata.org/pandas-docs/version/1.1/reference/io.html